desk/app/groups.hoon

```diff
@@ -1466,7 +1503,13 @@
           ==
         =?  cor  go-is-our-bloc
           (emit (pass-hark new-yarn))
-        go-core
+        =+  ships=~(tap in ships)
+        |-
+        ?~  ships  go-core
+        =.  go-core
+          =<  ?>(?=(%shut -.cordon.group) .)  ::NOTE  tmi
+          (go-activity %ask i.ships)
+        $(ships t.ships)
```

Comment:
hmm I wonder if we just want to support `(set ship)` for `ask` and similar activity